CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
-- ⚠️ `vast start` now prints the endpoint it is listening on on `stdout`.
+- ⚠️ `vast start` now prints the endpoint it is listening on on `stdout` when it
+  is invoked with the `--print-endpoint` option.
```

Comment:
```suggestion
- ⚠️ `vast start` now prints on `stdout` the endpoint it is listening on,
  provided the user added the `--print-endpoint` option.
```

Reply:
I took the liberty to remove "on `stdout`".